content/chainguard/chainguard-images/staying-secure/security-advisories/how-to-use/index.md

```diff
@@ -40,7 +40,7 @@ To follow along with these examples, you'll need the following tools installed.
 * [`chainctl`](/chainguard/chainctl/) — Chainguard's command-line interface tool. To install `chainctl`, follow our [installation guide](/chainguard/administration/how-to-install-chainctl/). 
 * [`jq`](https://jqlang.github.io/jq/) — `jq` is a command-line JSON processor that allows you to filter and manipulate streaming JSON data. Although it isn’t strictly necessary for the purposes of this guide, this tutorial includes commands that use `jq` to filter command output that would otherwise be difficult to read. You can install `jq` by following the instructions on [the project’s Download jq page](https://jqlang.github.io/jq/download/).
 
-Lastly, note that this guide includes examples involving a sample organization with a private Chainguard Registry named `example.com`. If you would like to follow along with your own private Chainguard Containers, be sure to change this where relevant to reflect your own setup. If you don't have access to a private Chainguard Registry, you can also follow along using Chainguard's public [Starter Containers](/chainguard/chainguard-images/about/images-categories/#starter-containers), but be aware that these are limited to only the `latest` or `latest-dev` tags. You can download public Starter Containers from the `cgr.dev/chainguard` registry, as in `cgr.dev/chainguard/go:latest`.
+Lastly, note that this guide includes examples involving a sample organization with a private  registry named `example.com`. If you would like to follow along with your own private Chainguard Containers, be sure to change this where relevant to reflect your own setup. If you don't have access to a private registry, you can also follow along using Chainguard's public [Starter Containers](/chainguard/chainguard-images/about/images-categories/#starter-containers), but be aware that these are limited to only the `latest` or `latest-dev` tags. You can download public Starter Containers from the `cgr.dev/chainguard` registry, as in `cgr.dev/chainguard/go:latest`.
```

Comment:
In this case, I think you might want to specify something like "a private registry provided by Chainguard"